core/src/main/java/com/microsoft/applicationinsights/internal/util/SSLOptionsUtil.java

```diff
@@ -0,0 +1,33 @@
+package com.microsoft.applicationinsights.internal.util;
+
+import com.google.common.base.Splitter;
+import com.google.common.collect.Iterables;
+import com.microsoft.applicationinsights.internal.logger.InternalLogger;
+
+import javax.annotation.Nullable;
+import java.util.Arrays;
+
+public class SSLOptionsUtil {
+
+    public static final String APPLICATION_INSIGHTS_SSL_PROTOCOLS_PROPERTY = "applicationinsights.ssl.protocols";
+
+    @Nullable
+    public static String[] getAllowedProtocols() {
+        final String rawProp = System.getProperty(APPLICATION_INSIGHTS_SSL_PROTOCOLS_PROPERTY);
+        final String[] defaultValue = {"TLSv1.2"};
```

Comment:
Java 11 uses TLSv1.3 by default
Java 8 uses TLSv1.2 by default
Maybe better to add special handling for Java 7 only?

Reply:
Two things here:
1. I wonder if we can add TLSv1.3 as another allowed protocol. As long as it doesn't fail on java 7, that should be fine.
2. Based on the guidance provided, we need to be explicit and allow to roll back to TLSv1 or TLSv1.1 if the customer needs that.

Reply:
I added TLSv1.3 and TLSv1.2 as possible options and now it validates each value to be sure there are no errors at runtime trying to parse a protocol name.